src/ExportHistory/Models/ExportJobTransitions.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.DurableTask.ExportHistory;
+
+/// <summary>
+/// Manages valid state transitions for export jobs.
+/// </summary>
+static class ExportJobTransitions
+{
+    /// <summary>
+    /// Checks if a transition to the target state is valid for a given export job state and operation.
+    /// </summary>
+    /// <param name="operationName">The name of the operation being performed.</param>
+    /// <param name="from">The current export job state.</param>
+    /// <param name="targetState">The target state to transition to.</param>
+    /// <returns>True if the transition is valid; otherwise, false.</returns>
+    public static bool IsValidTransition(string operationName, ExportJobStatus from, ExportJobStatus targetState)
+    {
+        return operationName switch
+        {
+            nameof(ExportJob.Create) => from switch
+            {
+                ExportJobStatus.Uninitialized when targetState == ExportJobStatus.Active => true,
```

Comment:
Does this mean `Uninitialized` is a defined state rather than having no state at all being `Uninitialized`?

And with `Completed` and `Failed`, is this path basically restarting/recreating the job?